Library/Homebrew/dev-cmd/pr-automerge.rb

```diff
@@ -71,7 +76,7 @@ def pr_automerge
 
     publish_args = ["pr-publish"]
     publish_args << "--tap=#{tap}" if tap
-    publish_args << "--autosquash" if args.autosquash?
+    publish_args << "--autosquash" unless args.no_autosquash?
```

Comment:
Shouldn't this be

```suggestion
    publish_args << "--no-autosquash" if args.no_autosquash?
```

Reply:
🤦 yes, that's correct. I really should be paying more attention here. Sorry 😅